indico/migrations/versions/20190227_1523_081d4c97060a_add_notification_sent_column_to_.py

```diff
@@ -0,0 +1,35 @@
+"""Add notification_sent column to Reservation
+
+Revision ID: 081d4c97060a
+Revises: 7aabedfb5e3a
+Create Date: 2019-02-27 15:23:31.582776
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '081d4c97060a'
+down_revision = '7aabedfb5e3a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('reservations', sa.Column('notification_sent', sa.Boolean(), nullable=False, server_default='false'),
```

Comment:
`end_notification_sent`, it's not very clear what kind of notification it is otherwise.

Reply:
I guess this wasn't fixed yet? Just saying because the conversation is marked as "resolved".